volume-providers/softlayer/utils/storage_utils.go

```diff
@@ -468,7 +473,14 @@ func ConvertToVolumeType(storage datatypes.Network_Storage, logger *zap.Logger,
 		volume.TargetIPAddresses = storage.IscsiTargetIpAddresses
 	} */
 
+	switch volume.VolumeType {
```

Comment:
is it really required to check volumeType as we can only check if that value is not null then get that vale else skip it. because we are already using json and omitempty tag

Reply:
done